### DISCUSSION TITLE
attachment datatype is not visible while creating table

### DISCUSSION THREAD

**Discussion by Athik321** [Budibase/budibase#14155] [Help] [answered]
When I create a table and want to add an attachment column, the attachment data type is not visible in the data type list.
Do I need to set up MinIO first to make attachment data type visible? I am using MySQL Database.

**Comment by mike12345567** [comment, marked as answer]
Hi @Athik321 - currently attachment column types are only available for the internal DB/tables - however external table support for attachment columns is on the roadmap.

**Reply by Athik321** [reply]
Okay, thanks for your time.
